aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -549,6 +555,9 @@ module aptos_token::token {
         maximum: u64,
         mutate_setting: vector<bool>
     ) acquires Collections {
+        // TODO: Validate against empty name and uri? May be even descrition as well if not optional.
```

Comment:
we allow empty string as name, URI and description now. If user don't want to put these fields, they can use empty string.